skore/src/skore/cli/cli.py

```diff
@@ -66,21 +68,28 @@ def cli(args: list[str]):
         "quickstart", help='Create a "project.skore" file and start the UI'
     )
 
+    if args is None:
+        args = sys.argv[1:]
     parsed_args: argparse.Namespace = parser.parse_args(args)
 
-    if parsed_args.subcommand == "launch":
-        __launch(
-            project_name=parsed_args.project_name,
-            port=parsed_args.port,
-            open_browser=parsed_args.open_browser,
-        )
-    elif parsed_args.subcommand == "create":
-        create(
-            project_name=parsed_args.project_name,
-            working_dir=parsed_args.working_dir,
-            overwrite=parsed_args.overwrite,
-        )
-    elif parsed_args.subcommand == "quickstart":
-        __quickstart()
-    else:
-        parser.print_help()
+    try:
+        if parsed_args.subcommand == "launch":
+            __launch(
+                project_name=parsed_args.project_name,
+                port=parsed_args.port,
+                open_browser=parsed_args.open_browser,
+            )
+        elif parsed_args.subcommand == "create":
+            create(
+                project_name=parsed_args.project_name,
+                working_dir=parsed_args.working_dir,
+                overwrite=parsed_args.overwrite,
+            )
+        elif parsed_args.subcommand == "quickstart":
+            __quickstart()
+        else:
+            parser.print_help()
+        return 0
+    except Exception:
+        raise
+        return 1
```

Comment:
Raise and return are mutually exclusive, the return will never be reached. Can you rollback this changes?

Reply:
Yep I clean up those. Sorry about that.